tools/federation/build/test/rdf4j-server/WEB-INF/rdf4j-http-server-servlet.xml

```diff
@@ -0,0 +1,150 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:aop="http://www.springframework.org/schema/aop"
+	xsi:schemaLocation="
+http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans-2.5.xsd
+http://www.springframework.org/schema/aop http://www.springframework.org/schema/aop/spring-aop-2.5.xsd">
+
+	<!-- RESOURCES -->
+	<bean id="messageSource"
+		class="org.springframework.context.support.ResourceBundleMessageSource">
+		<property name="basenames">
+			<list>
+				<value>org.eclipse.rdf4j.http.server.messages</value>
+				<value>org.eclipse.rdf4j.common.webapp.system.messages</value>
+				<value>org.eclipse.rdf4j.common.webapp.messages</value>
+			</list>
+		</property>
+	</bean>
+
+    <!-- Customized FedX: use custom init to avoid initialization of logging framework -->
```

Comment:
What's this for? Is this to avoid a lot of STDOUT output during test runs, or something else?

Reply:
From my memories this was required for my stripped-down rdf4j workbench to work properly w.r.t logging while executing the test. I think also it was to reduce dependencies (as I typically used log4j). Maybe this could now be change a bit again.

I also remember that there is an issue in the RDF4J tracker to make logging in the workbench more configurable, as this currently strictly requires logback.

For now I would suggest to keep this (as it is for testing only), if this is OK with you.